test/micro/org/openjdk/bench/java/nio/ByteBuffers.java

```diff
@@ -923,4 +923,9 @@ public double testDirectLoopGetDouble() {
         }
         return r;
     }
+
+    @Benchmark
+    public int testHashCode() {
+        return directByteBuffer.hashCode();
+    }
```

Comment:
The change is for the heap based ByteBuffer, it looks like you've added a benchmark for a direct ByteBuffer.

Reply:
Fixed